### PR TITLE
Add systemd unit file for rkt

### PR DESCRIPTION
**Pull request by caseydavenport**

### CODE REVIEW

File: docs/cni/rkt/systemd/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+sudo systemctl enable /etc/systemd/system/calico-node.service
+```
+
+Start the unit
+
+```
+sudo systemctl start calico-node.service
+```
+
+### 3. Check status
```
Comment (tomdee) [inline docs/cni/rkt/systemd/README.md]: It's worth mentionaing the machinectl stuff too

**Comment by tomdee** [comment]
Looks fine but this needs a link from somewhere

**Comment by tomdee** [comment]
@caseydavenport Shall I take on getting this merged?

**Comment by caseydavenport** [comment]
D'oh, I lost track of this one...

Just needs a link from somewhere right? Where would you like it?

**Comment by caseydavenport** [comment]
Or you can go ahead and finish it off.

**Comment by tomdee** [comment]
Rebased and merged manually.

**Pull request closed without merging**